server/db/courses.json

```
@@ -1,7 +1,7 @@
 [
     {
         "id": 1807272,
-        "title": "Khóa học tiếng Nhật Online cấp độ N1 (10 bài)",
+        "title": "Khóa học Online cấp độ N1 (10 bài)",
```

Bạn nghĩ chỗ này chỉ để "N1 (10 bài thôi á)" cho ngắn á. Các chỗ khác cũng vậy á
```suggestion
        "title": "N1 (10 bài)",
```